specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2022-08-11-preview/HybridCompute.json

```diff
@@ -0,0 +1,1637 @@
+            "schema": {
+              "$ref": "#/definitions/ExtensionValueListResult"
+            }
+          },
+          "default": {
+            "description": "Error in retrieving extension list",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "x-ms-examples": {
+          "GET a list of extensions": {
+            "$ref": "./examples/ExtensionMetadata_List.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.HybridCompute/operations": {
+      "get": {
+        "tags": [
+          "operations"
+        ],
+        "operationId": "Operations_List",
+        "description": "Gets a list of hybrid compute operations.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Hybrid Compute Provider Operations": {
+            "$ref": "./examples/Operations_List.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ExtensionValueListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/ExtensionValue"
+          },
+          "description": "The list of extension metadata",
+          "x-ms-identifiers": []
+        }
+      },
+      "description": "The List Extension Metadata response."
+    },
+    "ExtensionValue": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ExtensionValueProperties",
+          "description": "The single extension based on search criteria"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "description": "Describes a Extension Metadata"
+    },
+    "ExtensionValueProperties": {
+      "type": "object",
+      "properties": {
+        "requestId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The id of the Extension being received."
+        },
+        "versions": {
```

Comment:
>versions [](http://example.com/codeflow?start=9&length=8)

Should be singular unless it's a collection. If it's a collection it should be modeled as an array.

Reply:
as above

Reply:
changed to singular